brands/ghostery/branding/pref/firefox-branding.js

```diff
@@ -17,157 +19,116 @@ pref("app.update.url.manual", "https://www.ghostery.com/download/");
 // A default value for the "More information about this update" link
 // supplied in the "An update is available" page of the update wizard.
 pref("app.update.url.details", "https://www.ghostery.com/%LOCALE%/browser/notes");
-
-// Disable System Addon updates
-pref("extensions.systemAddon.update.url", "");
```

Comment:
this one has to stay - we currently don't use partial updates instead of system addon updates.

Reply:
OK. I've added this back. (Initially, I thought this would break extension updates.)

Reply:
Correction: After your comment, I added it [here](https://github.com/ghostery/user-agent-desktop/blob/4b345a9e49a85572ec406243de055db4ea8a597c/brands/ghostery/branding/pref/firefox-branding.js#L66-L68). But actually, I did retain this pref. It was further down in the file [here](https://github.com/ghostery/user-agent-desktop/blob/4b345a9e49a85572ec406243de055db4ea8a597c/brands/ghostery/branding/pref/firefox-branding.js#L121-L122).

I've made a note to remove the duplicate since this was incorrectly flagged as removed.